deployments/Dockerfiles/tendermint/build.sh

```diff
@@ -0,0 +1,2 @@
+/usr/bin/tendermint testnet --config /etc/tendermint/config-template.toml --v $1 --o . $2
+chown $3 -R /tendermint
```

Comment:
Fixes the files owned by root issue, the files will be owned by the default none root user